lib/ai/client.go

```diff
@@ -16,17 +16,24 @@ limitations under the License.
 
 package ai
 
+import "github.com/sashabaranov/go-openai"
+
 type Client struct {
-	apiURL string
+	svc openai.Client
 }
 
 func NewClient(apiURL string) *Client {
-	return &Client{apiURL}
+	return &Client{*openai.NewClient(apiURL)}
```

Comment:
Does this actually compile?

Reply:
Yes. It does. Declaration-order-init has been in the go spec for quite a while now.